docs/Developers/Git_Basics.rst

```diff
@@ -380,6 +380,11 @@ Now, to revert the file to that revision, run the command:
 Now, after the file gets reverted back to the required revision, commit the
 changes and (force)push to the remote.
 
+Sometimes while rebasing it might happen that you come across a Mid-Rebase
```

Comment:
Could be worded better, instead of `it might happen that you come across`, try something like `you may come across`

Thoughts?